core/tests/window/test_window.py

```diff
@@ -8,6 +8,7 @@
     assert_action_not_performed,
     assert_action_performed,
     assert_action_performed_with,
+    performed_actions,
```

Comment:
I'd prefer to see this in the form of an `assert_action_count` (or better still, an `assert_ action_performed(count=N)`).